apps/templates/content_templates.py

```diff
@@ -335,7 +349,8 @@ def get_item_from_template(template):
     """
     item = template.get('data', {})
     item[ITEM_STATE] = CONTENT_STATE.SUBMITTED
-    item['task'] = {'desk': template.get('template_desk'), 'stage': template.get('template_stage')}
+    if template.get('schedule_desk'):
```

Comment:
imo, we have to raise error over here if the `schedule_desk` is not available otherwise we have item context (desk/user) problem.